test-resources/py/delete_cleanup_str_in_list/configurations/rules.toml

```diff
@@ -115,6 +115,6 @@ query = """
 (#match? @list "\\\\[\\\\s*,?\\\\]")
 )
 """
-replace_node = "@assignment"
```

Comment:
The changes logic (below), now disallows wrong rules that reference non-existing variables